### PR TITLE
Adds APC and Transport Crewmen to the game

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request
Yippee APC
gets medical and (default) transport variants atm
mounts a cupola

## Changelog
:cl:
add: Adds APC and Transport Crewmen to the game
/:cl:

### CODE REVIEW

File: code/modules/vehicles/armored/interiors/ammo_rack.dm [hunk 1]
```diff
@@ -3,6 +3,7 @@
  */
 /obj/structure/ammo_rack //Parent type, only used as a template
 	icon = 'icons/obj/armored/3x3/tank_interior.dmi'
+	resistance_flags = RESIST_ALL
```
Comment (Lumipharon) [inline code/modules/vehicles/armored/interiors/ammo_rack.dm:6]: lol

File: _maps/interiors/apc_medical.dmm [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+/turf/closed/interior/apc/six,
+/area/interior/apc)
+"C" = (
+/turf/closed/interior/tank/door,
+/area/interior/apc)
+"D" = (
+/turf/open/interior/apc/fifteen,
+/area/interior/apc)
+"F" = (
+/obj/machinery/vending/MarineMed,
```
Comment (Lumipharon) [inline _maps/interiors/apc_medical.dmm:66]: wasn't being able to deploy a vendor with infinite meds something we didn't want?

**Review by Lumipharon — APPROVED** [review]

File: _maps/interiors/apc_medical.dmm (hunk 2 above)
Comment (TiviPlus) [reply _maps/interiors/apc_medical.dmm:66]: Kuro said so blame him

**Comment by Pockets-byte** [comment]
Transport suffers hugely from not being able to drag things into it

**Pull request merged**